packages/amplication-prisma-db/.env

```diff
@@ -4,3 +4,4 @@ POSTGRESQL_URL=postgresql://admin:admin@localhost:5432/${SERVICE_DB_NAME}
 POSTGRESQL_USER=admin
 POSTGRESQL_PASSWORD=admin
 SERVICE_DB_NAME=amplication
+PRISMA_CLIENT_OUTPUT_PATH= generated-prisma-client
```

Comment:
is that related to this PR?

Reply:
@yuval-hazaz  No, it's a new env var that is missing in the env file

Reply:
why not merging the missing variable into this branch?

Reply:
@yuval-hazaz, that's what I did, no??....

Reply:
please use a different PR